yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/object_type_mapping.psql

```diff
@@ -7,6 +7,7 @@ SELECT
         WHEN c.relkind = 'p' THEN 'partitioned table'
         WHEN c.relkind = 'i' THEN 'index'
         WHEN c.relkind = 'I' THEN 'partitioned index'
+        WHEN c.relkind = 'm' THEN 'materialized view'
```

Comment:
Just verify once that it is getting populated correctly in `export-dir/assessment/dbs/assessment.db`

Reply:
Had already checked. The tables are getting populated